dali/python/nvidia/dali/external_source.py

```diff
@@ -432,11 +432,16 @@ class ExternalSource():
     When batch is set to True, callables performance might especially benefit from increasing
     ``prefetch_queue_depth`` so that a few next batches can be computed in parallel.
 
-    Iterator or generator will be assigned to a single worker that will iterate over it.
+    .. note::
+        Iterator or generator will be assigned to a single worker that will iterate over it.
+        When ''batch'' is set to True multiple batches can be prepared in parallel for callables,
```

Comment:
'' or ``?
I am not sure if that makes a difference but we're using `` elsewhere.

Reply:
Fixed